lib/capistrano/tasks/rails_artifact.rake

```diff
@@ -1,26 +0,0 @@
-namespace :rails_artifact do
-  desc 'Check that the repo is reachable'
-  task :check do
-  end
-
-  desc 'Copy repo to releases'
-  task :create_release do
-    on release_roles :all do
-      archive_url = fetch(:rails_artifact_archive_location)
-      file_location = '/tmp/build-artifact-files/build-artifact.tar.gz'
-      execute :mkdir, '-p', File.dirname(file_location)
-      execute :rm, "-f #{file_location}"
-
-      execute :mkdir, '-p', release_path
-      execute :wget, "--no-check-certificate  -q -O '#{file_location}' '#{archive_url}'"
```

Comment:
perhaps that should be configurable... but, yea - probably shouldn't be the default